activities.py

```diff
@@ -447,6 +452,7 @@ def html_url(doc_name):
     'wicg.github.io': W3CParser,
     'dev.w3.org': W3CParser,
     'dvcs.w3.org': W3CParser,
+    'iot.mozilla.org': W3CParser,
```

Comment:
Check with @benfrancis, but I think that this is a one-off site and maybe not worth adding to this list.  Maybe a `-p` command-line option would be a better way to drive the selection of the parser for this sort of one-off.

Reply:
I'm hoping that the draft member submission at [iot.mozilla.org/wot](http://iot.mozilla.org/wot) will eventually be superseded by a W3C specification. That work is starting with [this Interest Group Note](http://w3c.github.io/wot/proposals/json-td/). But so far that only covers one area of our draft member submission, so we'll have to see how that work progresses. In the meantime we may continue to add to the draft member submission to reflect our latest thinking in its completeness.